aws-params-reader-policy/main.tf

```diff
@@ -30,6 +29,10 @@ data "aws_iam_policy_document" "policy" {
     actions   = ["kms:Decrypt"]
     resources = [data.aws_kms_alias.parameter_store_key.target_key_arn]
   }
+  statement {
+    actions   = ["ssm:DescribeParameters"]
+    resources = ["arn:aws:ssm:${var.region}:${data.aws_caller_identity.current.account_id}:parameter/*"]
```

Comment:
I briefly remember working on this with you. Was there no way to make this resource more specific? I remember we were having trouble getting this working with the `param_resources` because it was too specific for some reason. This would allow you to list all the parameters, which seems like too much.

Reply:
After doing some more digging, I think this is fine since describe parameters is a list option and the permission won't allow you to read the contents. According to the AWS console, its not even allowed to set a specific ARN with the following error: "The actions in your policy do not support resource-level permissions and require you to choose All resources". Can we document this in the statement so that this information is preserved?

Reply:
Noted! added a comment and an `sid` value